src/sentry/static/sentry/app/views/events/eventsChart.jsx

```diff
@@ -105,9 +105,9 @@ class EventsChart extends React.Component {
     yAxis: 'event_count',
   };
 
-  handleYAxisChange = value => {
```

Comment:
I think we need this bound here

Reply:
@billyvg I've re-checked all the bounds that I've removed and made changes in the latest commit.